modules/social_features/social_group/social_group.module

```diff
@@ -2805,3 +2805,15 @@ function social_group_preprocess_page(array &$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function social_group_preprocess_html(array &$variables):void {
```

Comment:
@agami4 Let's stick to one style and add a space before the `void`:
`function social_group_preprocess_html(array &$variables): void {`

Reply:
@Ressinel done